grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -0,0 +1,102 @@
+package org.pih.warehouse.inventory
+
+import grails.gorm.transactions.Transactional
+import grails.validation.ValidationException
+import org.grails.datastore.mapping.query.api.Criteria
+import org.hibernate.criterion.Order
+import org.hibernate.sql.JoinType
+import org.pih.warehouse.core.Location
+
+@Transactional
+class CycleCountService {
+
+    List<CycleCountCandidate> getCandidates(CycleCountCandidateFilterCommand command, String facilityId) {
```

Comment:
don't we need a filter for this field? Or has a ticket already been created for that?
![image](https://github.com/user-attachments/assets/b603b1e3-1358-4b2a-8315-16a9a9bd85f1)

Reply:
I skipped that part for now, because those fields were meant to "include later". For now negative item count is hardcoded as `null`, and I am not sure what this actually will be later.